app/src/lib/editors/darwin.ts

```diff
@@ -217,6 +225,7 @@ export async function getAvailableEditors(): Promise<
     typoraPath,
     codeRunnerPath,
     slickeditPath,
+    xCodePath,
```

Comment:
Minor, but could this be `xcodePath` to avoid "c" having any special capitalization rules?